lib/events.js

```diff
@@ -10,4 +10,6 @@ function trigger(pusher, channels, eventName, data, socketId, callback) {
   pusher.post({ path: '/events', body: event }, callback);
 }
 
-exports.trigger = trigger;
+exports.triggerBatch = function(pusher, batch, callback) {
+	pusher.post({ path: '/batch_events', body: { batch: batch } }, callback);
```

Comment:
if you want to allow passing arbitrary "data" attributes, they need to be encoded to JSON before posting